idea/testData/intentions/convertLambdaToMultiLine/simple4.kt.after

```diff
@@ -0,0 +1,6 @@
+// WITH_RUNTIME
+fun test(list: List<String>) {
+    list.forEach { item ->
+        /* aaa */ println(item); println(item) /* bbb */
```

Comment:
Let's move every statement inside lambda to its own line in such cases as we already have multi-line lambda